pages/manage_and_operate/iam/iam-logs-forwarding/guide.en-gb.md

```diff
@@ -0,0 +1,222 @@
+---
+title: OVHcloud Account logs Forwarding to LDP
+excerpt: "Forwarding OVHcloud Account logs to Logs Data Platform"
+updated: 2023-10-23
+---
+
+ 
+
+## Objective
+
+In this guide, you will learn how to transform your OVHcloud Account events as actionable data through Logs Data Platform, a fully managed solution that helps you store, archieve, query and visualize your logs.  
+To discover Logs Data Platform before continuing this guide, please refer to [this documentation](/pages/manage_and_operate/observability/logs_data_platform/getting_started_introduction_to_LDP).
+
```

Comment:
Maybe a small warning here: 

"This action will inject data into Logs Data Platform and could generate fees."

Reply:
Warning is at the begining of the "Enable logs forwarding" section